docs/how-to-guides/deploy.md

```diff
@@ -6,22 +6,78 @@ id: deploy
 
 ## Overview
 
-This guide shows you how to deploy any realm or package to the Gno chain. Deployment is be done by utilizing `gnokey`'s `maketx addpkg` API.
+This guide shows you how to deploy any realm or package to the Gno chain. It will
+show you how to:
+- Deploy Gno code in your browser via the Playground, 
+- Deploy Gno code from your local machine using `gnokey`'s `maketx addpkg` API.
```

Comment:
```suggestion
- deploy Gno code from your local machine using `gnokey`'s `maketx addpkg` API.
```

Reply:
suggesting lowercase as it looks like its a series of steps and a continuation of the previous sentence, with the comma after the first point.